app/posts/find-teacher-training/2022-02-11-updating-international-candidates-section.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Updating the ‘International candidates’ section
+description: Why we have updated and renamed this section
+date: 2022-02-11
+screenshots:
+  items:
+    - text: International candidates - sponsorship may be available
+      src: find-sponsorship-available.png
+    - text: International candidates - sponsorship not available
+      src: find-sponsorship-not-available.png
+related:
+  items:
+    - text: GitHub pull request making the change
+      href: https://github.com/DFE-Digital/find-teacher-training/pull/1138
+    - text: Design history entry for previous change
+      href: /find-teacher-training/new-international-students-section/
+---
+
+In August last year, we [added a new ‘International students’ section](/find-teacher-training/new-international-students-section/) as a place where we could explain things that people applying from outside the UK need to know, including whether the course offered visa sponsorship or not.
+
+We have renamed this section ‘International candidates’ for more consistency with the ‘candidate’ term we use elsewhere, and because users are not students at the point of applying, and may also apply for a salaried course where the term ‘student’ is less applicable.
```

Comment:
```suggestion
We have renamed this section ‘International candidates’ because:

* it's more consistent with ‘candidate’, which we use throughout Apply for teacher training
* users are not students at the point of applying
* candidates may apply for a salaried course where the term ‘student’ is less applicable
```

Reply:
I thought these 2 reasons were the strongest of the 3. Wasn't convinced by the other one so much!

Reply:
Fair! I think all 3 were variously mentioned at different points, but agree that those 2 are the strongest.